src/frontend/src/components/FavoriteButton.tsx

```diff
@@ -0,0 +1,78 @@
+import { useTranslation } from "react-i18next";
+
+import StarIcon from "assets/icons/star.svg?react";
+import Button from "components/base/Button/Button";
+import { useNavigate } from "react-router";
+import { updateUserProfile } from "src/reducers/userProfile/userProfileThunk";
```

Comment:
Let's remove `src/`

Reply:
No worries, I forgot about it before as well. I think we could update the tsconfig and vite config in the future so that auto imports won't add the `src` piece. All good.

Reply:
That'd be nice! Adding it to my list of incoming tech debt issues 👍